hbase-server/src/main/java/org/apache/hadoop/hbase/util/LoadTestKVGenerator.java

```diff
@@ -106,9 +107,11 @@ private static byte[] getValueForRowColumn(int dataSize, byte[]... seedStrings)
         seed += bytesString.hashCode();
       }
     }
-    Random seededRandom = new Random(seed);
     byte[] randomBytes = new byte[dataSize];
-    seededRandom.nextBytes(randomBytes);
+    synchronized (SHARED_RANDOM) {
```

Comment:
IMO you should just add an annotation to ignore this.

Spotbugs doesn't grasp what is being done here.

Reply:
Yes you are right, might cause slowness. Let me revert and annotate this one..